cluster/examples/kubernetes/ceph/csi/example/rbd/get.bash

```diff
@@ -0,0 +1,2 @@
+key=$(pod=$(kubectl get pod  -n rook-ceph -l app=rook-ceph-operator  -o jsonpath="{.items[0].metadata.name}"); kubectl exec -ti -n rook-ceph ${pod} -- bash -c "ceph auth get-key client.admin -c /var/lib/rook/rook-ceph/rook-ceph.config | base64")
```

Comment:
What's this file do? I don't see it referred to elsewhere in the doc?

Reply:
Yeah, it shouldn't be here, I'll remove it thanks!